jenkins/run-alembic-migrations-check

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash -e
 
+PB_CONTAINER_REG=${PB_CONTAINER_REG:-$(<${HOME}/.config/pbench/ci_registry.name)}
```

Comment:
I think this is really good.  I suspect that this idiom is going to crop up again for other uses....